5019.bugfix.rst

```diff
@@ -0,0 +1 @@
+Pin sanic patch version instead of minor version. Fixes sanic ``_run_request_middleware()`` error.
```

Comment:
@erohmensing this should be in `changelog/` not in the root dir

Reply:
ach, good catch!